datanode/api/event_observer.go

```diff
@@ -16,13 +16,14 @@ import (
 	"context"
 	"time"
 
+	"google.golang.org/grpc/codes"
+
```

Comment:
Ususally we order the package like so:
- std
- external
- vega

(although while I've always done that, I realised recently that goimports does like you do? which is weird as I've always used goimport and it used to be the behaviour I describe I'm sure???)

Reply:
but isn't it like that?

```
- std:
"context"
 "time"

- external:
"google.golang.org/grpc/codes"

- vega:
"code.vegaprotocol.io/vega/core/events"
"code.vegaprotocol.io/vega/datanode/metrics"
"code.vegaprotocol.io/vega/datanode/subscribers"
"code.vegaprotocol.io/vega/logging"
protoapi "code.vegaprotocol.io/vega/protos/vega/api/v1"
eventspb "code.vegaprotocol.io/vega/protos/vega/events/v1"
```

Reply:
oh I was misraken in what I said:
-std
-vega
-external